### ISSUE TITLE
Overlays cause Z-Fighting on some designs.

### ISSUE THREAD

**Issue by Flatgub** [PianoManu/BlockCarpentry#56]
![2021-08-31_15 00 46](https://user-images.githubusercontent.com/6888736/131448353-16918546-9939-4bfc-920d-dde0f2d6d29f.png)
On blocks like doors which have a glass portion, overlays cause visual Z-Fighting issues. This doesn't happen when the design used is completely opaque.
![2021-08-31_15 43 01](https://user-images.githubusercontent.com/6888736/131448500-62ab347c-86fa-4951-8599-0625f78b8d4e.png)

**Comment by PianoManu** [comment]
Yeah, I've already noticed that some time ago but I didn't have the courage to fix that then... Looks like I have to fix it now 😄 

Regards, Manu

**Comment by PianoManu** [comment]
![image](https://user-images.githubusercontent.com/59123244/201763339-706847ec-d91a-4cc2-9007-362d019dfade.png)

I know this issue is open for more that a year, but finally I had some time... It's getting better, at least for trapdoors. It's a lot of work and pretty complex, though

Regards, Manu

**Comment by vgrynch** [comment]
Adding my here, more visually noticeable:
![image](https://github.com/PianoManu/BlockCarpentry/assets/46818018/0e26d313-f266-467d-a163-6f0f9d033362)
